### DISCUSSION TITLE
Build failure with gobject-introspection 1.66.1

### DISCUSSION THREAD

**Discussion by xypron** [fwupd/fwupd#2864] [Q&A]
**Describe the bug**

Building fails with 
subprojects/libxmlb/src/meson.build:105:2: ERROR: Dependency "gobject-introspection-1.0" not found, tried pkgconfig and cmake

Debian package gobject-introspection (1.66.1-1) is installed.

**Steps to Reproduce**

On Debian Bullseye:
```
mkdir build
cd build
meson ..
```

**Expected behavior**

Build with current packages.

**fwupd version information**
Please provide the version of the daemon and client.
```shell
$ fwupdmgr --version
```

Please note how you installed it (`apt`, `dnf`, `pacman`, source, etc):

**fwupd device information**
Please provide the output of the fwupd devices recognized in your system.

```shell
$ fwupdmgr get-devices --show-all-devices
```

**Additional questions**
- Operating system and version:
Debian Bullseye

- Have you tried rebooting?
Not relevant for building

- Is this a regression?
No clue

**Comment by superm1** [comment]
You can either turn off introspection support with the meson option or you need to install the introspection package for libxmlb in advance. It's contained in the debian archives.

**Reply by superm1** [reply]
Specifically the package name is `gir1.2-xmlb-1.0`

**Comment by monochromec** [comment]
In Bullseye the corresponding pkg-config file is contained in the package libgirepository1.0-dev.
